tests/unit/test_review_model.py

```diff
@@ -0,0 +1,36 @@
+from pyosmeta.parse_issues import ReviewModel
+
+# We could setup some example data using fixtures and a conf.py
+# Once we have a better view of the test suite.
+example = {
```

Comment:
we may end up creating fixtures for some of these things that we can reuse. but for now i just added a variable object to the test so we have a test for aliases